### ISSUE TITLE
Could not be loaded due to missing `../data/web-api.json`

### ISSUE THREAD

**Issue by favna** [Gerrit0/typedoc-plugin-mdn-links#14]
With version 3.1.2 I get the following error:

```
[error] The plugin typedoc-plugin-mdn-links could not be loaded.
[error] Error: Cannot find module '../data/web-api.json'
Require stack:
- F:\sapphiredev\website-main\node_modules\typedoc-plugin-mdn-links\dist\webApi.js
- F:\sapphiredev\website-main\node_modules\typedoc-plugin-mdn-links\dist\index.js
- F:\sapphiredev\website-main\node_modules\typedoc\dist\lib\utils\plugins.js
- F:\sapphiredev\website-main\node_modules\typedoc\dist\lib\utils\index.js
- F:\sapphiredev\website-main\node_modules\typedoc\dist\lib\models\comments\comment.js
- F:\sapphiredev\website-main\node_modules\typedoc\dist\lib\models\reflections\abstract.js
- F:\sapphiredev\website-main\node_modules\typedoc\dist\lib\models\reflections\index.js
- F:\sapphiredev\website-main\node_modules\typedoc\dist\lib\models\index.js
- F:\sapphiredev\website-main\node_modules\typedoc\dist\lib\converter\context.js
- F:\sapphiredev\website-main\node_modules\typedoc\dist\lib\converter\index.js
- F:\sapphiredev\website-main\node_modules\typedoc\dist\lib\application.js
- F:\sapphiredev\website-main\node_modules\typedoc\dist\index.js
- F:\sapphiredev\website-main\node_modules\docusaurus-plugin-typedoc\dist\plugin.js
- F:\sapphiredev\website-main\node_modules\docusaurus-plugin-typedoc\dist\index.js
- F:\sapphiredev\website-main\node_modules\@docusaurus\utils\lib\moduleUtils.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1048:15)
    at Module._load (node:internal/modules/cjs/loader:901:27)
    at Module.require (node:internal/modules/cjs/loader:1115:19)
    at require (node:internal/modules/helpers:130:18)
    at Object.<anonymous> (F:\sapphiredev\website-main\node_modules\typedoc-plugin-mdn-links\dist\webApi.js:4:16)
    at Module._compile (node:internal/modules/cjs/loader:1241:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1295:10)
    at Module.load (node:internal/modules/cjs/loader:1091:32)
    at Module._load (node:internal/modules/cjs/loader:938:12)
    at Module.require (node:internal/modules/cjs/loader:1115:19)
    at require (node:internal/modules/helpers:130:18)
    at Object.<anonymous> (F:\sapphiredev\website-main\node_modules\typedoc-plugin-mdn-links\dist\index.js:6:18)
    at Module._compile (node:internal/modules/cjs/loader:1241:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1295:10)
    at Module.load (node:internal/modules/cjs/loader:1091:32)
    at Module._load (node:internal/modules/cjs/loader:938:12)
    at Module.require (node:internal/modules/cjs/loader:1115:19)
    at require (node:internal/modules/helpers:130:18)
    at loadPlugins (F:\sapphiredev\website-main\node_modules\typedoc\dist\lib\utils\plugins.js:14:28)
    at async Application.bootstrapWithPlugins (F:\sapphiredev\website-main\node_modules\typedoc\dist\lib\application.js:194:13)
    at async generateTypedoc (F:\sapphiredev\website-main\node_modules\docusaurus-plugin-typedoc\dist\plugin.js:69:17)
```

I think you need to update the `files` array in package.json:

https://github.com/Gerrit0/typedoc-plugin-mdn-links/blob/fb68a8de18afd08273c761ab731271fc57991ee5/package.json#L17-L21

These are the published files:
![image](https://github.com/Gerrit0/typedoc-plugin-mdn-links/assets/4019718/ed34ace2-d19c-41a9-b098-46adc7c64e35)

**Comment by Gerrit0** [comment]
Sorry about that!

**Comment by Gerrit0** [comment]
Fixed in 3.1.3
